proto/kvrpcpb.proto

```diff
@@ -916,6 +920,11 @@ enum APIVersion {
     V2 = 2;
 }
 
+message TidbSource {
```

Comment:
How about `QueryCtx` or `StmtCtx`? Our goal is to establish a connection between the execution flow of transactions or statements.
`connection_id` may not be globally unique, to clarify which query the request belongs to something like `stmt_id` or `uuid` is needed but it looks a bit complex.